src/main/java/com/raoulvdberge/refinedstorage/gui/grid/GuiGrid.java

```diff
@@ -201,7 +201,11 @@ private int getRows() {
     }
 
     private boolean isOverSlotWithItem() {
-        return grid.isConnected() && isOverSlot() && slotNumber < STACKS.size();
+        return isGridActive() && isOverSlot() && slotNumber < STACKS.size();
+    }
+
+    private boolean isGridActive() {
```

Comment:
Is this method really needed? Can't we just call `grid.isActive()` immediatally?